buildSrc/src/test/java/org/elasticsearch/gradle/BwcVersionsTests.java

```diff
@@ -279,6 +281,11 @@
         sampleVersions.put("7.1.0", asList("7_1_0", "7_0_0", "6_7_0", "6_6_1", "6_6_0"));
     }
 
+    @BeforeClass
+    public static void setupAll() {
+        Assume.assumeFalse(Architecture.current() == Architecture.AARCH64);
```

Comment:
For now we just ignore the unit tests on aarch64 as we filter out there anything < 7.12